oeps/oep-0005/decisions/0004-To-Make-Or-Not-To-Make.rst

```diff
@@ -0,0 +1,65 @@
+To Make or Not to Make
+======================
+
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+For Decentralized Devstack, an common interface is necessary to enable a OpenedX Developer to add contributions to the many independent services in Open edX codebase.
+
+Options
+~~~~~~~
+
+1. The current Devstack uses make targets as an basic interface for developers.
+
+2. Use docker-compose commands directly
```

Comment:
I would add "Use docker-compose commands directly _for process control_"

There are other operations I think we should support as part of a good developer experience; perhaps those should be their own ADRs, but this doc should be clear that may still be a `make` (or other) interface and that it is _only_ addressing the process control aspects.

Also this document appears to assume that _all_ our projects use docker for process control, which is not currently true.

Further, if we ever have projects that use something else or we want to smoothly migrate to something else, we would need to go back wrap everything to maintain consistency; otherwise it's not just one extra tool developers need to learn, it's N tools and the overhead of which projects use which.

Reply:
Changing this to: "Use docker-compose commands directly for managing images and containers"